tabbar.cpp

```diff
@@ -51,7 +51,8 @@ TabBar::TabBar(QWidget *parent) :
 
     ProjectExplorer::SessionManager *sm = ProjectExplorer::SessionManager::instance();
     connect(sm, &ProjectExplorer::SessionManager::sessionLoaded, [this, em]() {
-        for (Core::DocumentModel::Entry *entry : Core::DocumentModel::entries())
+        Core::DocumentModel::Entry *entry = nullptr;
+        foreach (entry, Core::DocumentModel::entries())

```

Comment:
What's the reason for out of loop scope `entry`? The following works with pre C++11 compilers.
```cpp
foreach (Core::DocumentModel::Entry *entry, Core::DocumentModel::entries())
```